pkg/project/common/taskw

```diff
@@ -115,4 +115,6 @@ fi
 export TASK_COLOR_GREEN=35
 export TASK_TEMP_DIR="${AEM_DIR}/tmp/task"
 
+./aemw config export -f aem.env >/dev/null 2>&1
```

Comment:
to be removed; after all I don;t like sync mechanism;

I will use env file to be loaded separately by AEMC and task tool.

Reply:
done